openmetadata-ui/src/main/resources/ui/cypress/constants/tagsAddRemove.constants.js

```diff
@@ -19,7 +19,7 @@ export const TAGS_ADD_REMOVE_ENTITIES = [
     serviceName: 'sample_data',
     fieldName: 'SKU',
     tags: ['PersonalData.Personal', 'PII.Sensitive'],
-    isTable: true,
+    separate: true,
```

Comment:
Once we have it at all the places we will remove this correct?

Reply:
yes, we will remove it!